descartes_light/core/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ add_library(
   src/solvers/ladder_graph/ladder_graph_dag_search.cpp)
 target_link_libraries(${PROJECT_NAME} PUBLIC Eigen3::Eigen OpenMP::OpenMP_CXX console_bridge::console_bridge)
 target_cxx_version(${PROJECT_NAME} PUBLIC VERSION ${DESCARTES_CXX_VERSION})
-target_compile_options(${PROJECT_NAME} PUBLIC -std=c++${DESCARTES_CXX_VERSION})
+target_compile_options(${PROJECT_NAME} PRIVATE -std=c++${DESCARTES_CXX_VERSION})
```

Comment:
I believe this line can be removed because the cxx version should be correctly set by the use of [target_cxx_version](https://github.com/ros-industrial/ros_industrial_cmake_boilerplate#set-target-cxx-version). Does it work as expected if it is removed?

Reply:
I checked it and the line can indeed be removed. Will you modify the pull request?

Reply:
Just modified it.